met-web/src/components/engagement/form/EngagementWidgets/Video/Form.tsx

```diff
@@ -143,7 +145,19 @@ const Form = () => {
                             spacing={2}
                         >
                             <Grid item xs={12}>
-                                <MetLabel>Description</MetLabel>
+                                <MetLabel>Title (Optional)</MetLabel>
+                                <ControlledTextField
+                                    name="title"
+                                    variant="outlined"
+                                    label=" "
```

Comment:
Is there some other labelling happening here for screen readers?

Reply:
Thanks Alex, I'll create some labels for screen readers :)